config/base/kustomization.yaml

```diff
@@ -9,6 +9,7 @@ resources:
 - serviceaccount.yaml
 - validation-service.yaml
 - kong-ingress-dbless.yaml
+- prometheus
```

Comment:
Like I said: let's not include prometheus in `base` because that will make the `base` kustomization fail if the user does not have Prometheus Operator installed in their cluster.

I think that it would be OK to have a separate kustomization for prometheus integration, and tell the user to apply that kustomization separately from KIC.

Reply:
I take the input "that will make the base kustomization fail if the user does not have Prometheus Operator installed in their cluster." 👍 

Follow the seperate kustomization idea, does following file hierarchy look good @mflendrich @rainest 

config/prometheus/kustomization.yaml
config/prometheus/servicemonitor.yaml
config/prometheus/crd.yaml

Reply:
The hierarchy looks good with the following remarks:
- we'll need to remove the CRD (see the other comment)
- we have a PodMonitor and not a ServiceMonitor, so the servicemonitor.yaml file should probably be renamed

Reply:
changed accordingly b7c4fbb5d58fd7878a29a0251f69d11ac59c3a62